index.html

```diff
@@ -17,170 +17,189 @@
     <header class="header" id="home">
         <div class="wrapper">
             <nav class="header__nav">
-                <div class="header__nav--logo">
-                    <img src="images/logo.svg" alt="">
+                <div class="header__logo">
+                    <img src="images/logo.svg" alt="logo firmy nova">
                 </div>
-                <div class="header__nav--links">
+                <div class="header__links">
                     <a href="#team">The Team</a>
                     <a href="#pricing">Pricing</a>
                     <a href="#features">Features</a>
                 </div>
             </nav>
             <div class="header__content">
-                <div class="header__content--left">
-                    <h1 class="header__content--left-title">Beautiful Free Nova template</h1>
-                    <p class="header__content--left-txt">A top notch premium quality template for your next web project.
+                <div class="header__left-side">
+                    <h1 class="header__title">Beautiful Free Nova template</h1>
+                    <p class="header__txt">A top notch premium quality template for your next web project.
                     </p>
-                    <button class="header__content--left-btn">Download FREE!</button>
+                    <button class="header__btn">Download FREE!</button>
                 </div>
-                <div class="header__content--right">
-                    <img class="header__content--right-img" src="images/screen.png" alt="website layout">
+                <div class="header__right-side">
+                    <img class="header__img" src="images/screen.png" alt="website layout">
                 </div>
             </div>
         </div>
     </header>
 
     <section class="features" id="features">
-        <h2 class="section_header features__header">features</h2>
+        <h2 class="features__header section-header">features</h2>
         <div class="wrapper">
-            <div class="features__content">
-                <div class="features__content--card">
-                    <div class="features__content--card-circle"><img src="./images/responsive.svg" alt=""></div>
-                    <span class="features__content--card-title">fully responsive</span>
-                    <p class="features__content--card-txt">looks amazing on any device:
+            <ul class="features__cards">
+                <li class="features__card">
+                    <figure class="features__circle"><img src="./images/responsive.svg" alt=""></figure>
+                    <span class="features__title">fully responsive</span>
+                    <p class="features__txt">looks amazing on any device:
```

Comment:
Jeśli zaczyna całość za bardzo się zagnieżdżać to właśnie można użyć nowego bloku (pisałem o tym wyżej)